content/attacks/posts/2023-05-18-CryptoVault.md

```diff
@@ -0,0 +1,32 @@
+---
+date: 2023-05-18
+custodians: CryptoVault
+categories: Custodian, Smart contract
+title: CryptoVault Suffers Smart Contract Vulnerability Leading to $50 Million Loss
+---
+
+##Summary
+
+On May 18, 2023, CryptoVault, a prominent custodian in the blockchain space, experienced a significant security breach resulting in a loss of $52.6 million worth of cryptocurrencies according to US investors reports. The attack was attributed to a vulnerability in one of CryptoVault's smart contracts, which allowed the attacker to exploit the system and transfer the funds to an unauthorized account. The incident highlights the importance of rigorous smart contract auditing and security practices within the blockchain industry.
```

Comment:
fact checking bot i right. if you don't know how much they lost, don't write it or change the language to "at least"